### PR TITLE
Travis build

### PR DESCRIPTION
**Pull request by shijinabraham-leapyear**

### CODE REVIEW

**Comment by shijinabraham-leapyear** [comment]
Build will be at https://travis-ci.org
Currently I am changing the sub-module url in .gitmodules.

File: .travis.yml [hunk 1]
```diff
@@ -11,8 +11,10 @@ env:
 cache:
   directories:
   - $HOME/.stack
-#  - /nix/store
 before_install:
+
+ - cd inline-java && git log origin/leapyear-patchset | grep $(git rev-parse HEAD) && cd ..
+ 
  - nix-build --no-out-link -A stack    ./nixpkgs.nix
  - nix-build --no-out-link -A openjdk7 ./nixpkgs.nix
```
Comment (robinbb) [inline .travis.yml]: This line is unnecessary and wasteful. It became so when we switched to JDK8. Simply remove this line.

**Comment by robinbb** [comment]
Please also squash all of the changes in this PR into a single commit. (`git rebase`)

File: .travis.yml (hunk 1 above)
Comment (shijinabraham-leapyear) [inline .travis.yml]: removed the line

**Comment by shijinabraham-leapyear** [comment]
Can you squash merge this? Or I have to open a new PR from the rebased branch

**Comment by robinbb** [comment]
@shijinabraham-leapyear I think that you should be able to submit a single commit on top of leapyear-patchset as the PR by constructing that and then force-pushing to your branch. @mihaimaruseacleapyear is good with such things. Perhaps he will help. Then, I will "rebase and merge".

**Comment by robinbb** [comment]
Please merge both commits into one.

**Comment by shijinabraham-leapyear** [comment]
done.

**Comment by robinbb** [comment]
@shijinabraham-leapyear I think a mistake was made in the rebasing that you did (not necessarily your fault). I propose simply copy-and-pasting the desired changes from here into a fresh clone of leapyear-patchset, and then submitting that one commit so that I can rebase and merge. Use different PR.

**Pull request closed without merging**